src/app/visual/date-range-picker/date-range-picker-visual.component.ts

```diff
@@ -48,7 +48,6 @@ export class DateRangePickerVisualComponent implements OnInit {
     });
 
     this.reactiveRange.statusChanges
-      .distinctUntilChanged()
```

Comment:
Why were these distinct checks removed?

Reply:
They were swallowing multiple calls from the emitter. I wanted to verify that the events were only emitting once, instead of multiple times.